test/CAS/Tests/AuthenticationTest.php

```diff
@@ -132,13 +136,13 @@ protected function tearDown()
      */
     public function testRedirect()
     {
-        ob_start();
+        \ob_start();
```

Comment:
Sorry if I didn't catch this one before, but do we really need explicit global namespace for `ob_start` and `ob_end_clean`? It's reasonable if the test class wants to override it.

I think it's OK for `\Exception` since it needs to be broad.

Reply:
> Sorry if I didn't catch this one before, but do we really need explicit global namespace for ob_start and ob_end_clean?

Good catch. Nope, for functions this is not necessary. It is only necessary for classes.

I've fixed this in the latest revision.